### PR TITLE
perfecto-upload 0.0.3

### PR DESCRIPTION
**Pull request by sreevatsa-ramya**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by trapacska** [comment]
Thanks @sreevatsa-ramya checking. Should be ok. 🙂

File: step-info.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+maintainer: community
```
Comment (trapacska) [inline step-info.yml:1]: please move this file to this path in this PR: `steps/perfecto-upload/step-info.yml`
currently it is in the root of the repo, that is why the CI failed.

**Comment by sreevatsa-ramya** [comment]
Hi,

I have done the asked changes.

BR,
Sree

File: step-info.yml (hunk 1 above)
Comment (sreevatsa-ramya) [reply step-info.yml]: Have done.

**Comment by trapacska** [comment]
@sreevatsa-ramya checking. Thanks!

File: steps/perfecto-upload/0.0.3/step.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+summary: |
+  Upload APK/IPA files to Perfecto cloud repository
+description: |
+  Upload your build files to Perfecto for remote device testing. Upload your IPA/APK file to Perfecto repository
+website: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+source_code_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+support_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/issues
+published_at: 2020-06-15T19:08:15.426826+05:30
+source:
+  git: https://sreevatsa-ramya:Sreesmrithi123%24@github.com/PerfectoCode/Bitrise_Perfecto_FileUpload.git
```
Comment (trapacska) [inline steps/perfecto-upload/0.0.3/step.yml:11]: @sreevatsa-ramya please update the url

Comment (sreevatsa-ramya) [reply steps/perfecto-upload/0.0.3/step.yml]: Updated.

**Pull request merged**